markdown/manual.md

```diff
@@ -1,220 +1,245 @@
+### Reporting and Resolution of Violations
+Any employee who believes they have been harassed or believes they have witnessed another member of our organization violate the terms of this document should report the situation immediately to one of the following members of management: Executive Director, Treasurer or Chairman of the Board of Directors. If an employee makes a report to one of these persons and the employee does not receive a satisfactory response, the employee shall report the situation to one of the other designated persons to receive complaints.
 
+The organization will investigate all such reports as confidentially as possible. Adverse action will not be taken against an employee because he or she, in good faith, reports or participates in the investigation of a violation of this policy. Violations of this policy are not permitted and may result in disciplinary action, up to and including discharge.
 
-### Drug & Alcohol-Free Workplace Statement
-The unlawful or improper presence or use of controlled substances or alcohol in the workplace presents a danger to all employees.  Employees are prohibited from reporting to work or working while using illegal or unauthorized substances.
+### Drug & Alcohol Policy
+The unlawful or improper presence or use of controlled substances or alcohol in the workplace presents a danger to all employees.  All members of the organization are prohibited from reporting to work or working while using illegal or unauthorized substances.
+
+Our organization recognizes that drug or alcohol abuse are serious medical conditions and should be treated as such. Therefore, we shall provide all appropriate organizational support and assistance to those within the organization who request it.  We will also seek out and retain a employer-sponsored health insurance plan that provide drug and alcohol treatment services.
 
 ### Conflict of Interest
-A company's reputation for integrity is its most valuable asset and is directly related to the conduct of its employees.  Therefore, employees must never use their positions with the company for private gain, to advance personal interests or to obtain favors or benefits for themselves, member of their families or any other individuals, corporations of business entities.
+Employees must never use their positions with the organization for private gain, to advance personal interests or to obtain favors or benefits for themselves, member of their families or any other individuals, corporations of business entities. Furthermore, employees should avoid situations where a *perception* of a conflict of interest could be created.
+
+Should you encounter a situation that suggests a conflict of interest, please refer the situation the Executive Director, who will in turn seek a determination from the Board of Directors as to whether a conflict exists, and if it exists, whether it is an allowable conflict.
```

Comment:
@batemapf really good here. Thank you.  I like "perception or appearance, too, but perception alone works too.

Reply:
👍